rmw_connextdds_common/src/common/rmw_impl_waitset_std.cpp

```diff
@@ -550,6 +564,28 @@ RMW_Connext_WaitSet::wait(
   return RMW_RET_OK;
 }
 
+void
+RMW_Connext_StatusCondition::on_inconsistent_topic(
+  const struct DDS_InconsistentTopicStatus * status)
+{
+  update_state(
+    [this, status]() {
+      this->update_status_inconsistent_topic(status);
+    }, true /* notify */);
+}
+
+void
+RMW_Connext_StatusCondition::update_status_inconsistent_topic(
+  const struct DDS_InconsistentTopicStatus * status)
+{
+  this->status_inconsistent_topic = *status;
+  this->triggered_inconsistent_topic = true;
+
+  this->status_inconsistent_topic.total_count_change = this->status_inconsistent_topic.total_count;
+  this->status_inconsistent_topic.total_count_change -=
+    this->status_inconsistent_topic_last.total_count;
```

Comment:
I am not sure if this calculation is required to be done in RMW, it seems that we can just store the data from DDS w/o processing?

besides, `total_count_change` also needs to count `undiscovered` inconsistent topic? i was thinking that it just incremental number to `discover` inconsistent topics only.

Reply:
Let me test this out and see if we can just copy it out of status directly.

Reply:
And yeah, we can.  So this is removed in 8f7660eefd6bb159944e9881fe1459e947e23d49